### PR TITLE
Revert "Revert "Merge pull request #403 from ava-labs/verify-with-tru…

### PR DESCRIPTION
**Pull request by yulin-dong**
…ffle-verify""

This reverts commit 34272b915e06c179325c0c534c1f80f3987c9228.

### CODE REVIEW

**Comment by yulin-dong** [comment]
@Julian-dev28 please remove reference to verify-smart-contracts-with-sourcify-truffle.md and add reference to the new doc verify-smart-contracts-with-truffle-verify.md in the proper place.

**Comment by yulin-dong** [comment]
https://github.com/ava-labs/avalanche-docs/pull/414 has been merged, we can move forward with this PR.

File: docs/build/tutorials/smart-contracts/verify-smart-contracts.md [hunk 1]
```diff
@@ -8,17 +8,20 @@ If you have issues, contact us on [Discord](https://chat.avalabs.org).
 
 ## Steps
 
-Navigate to the _Code_ tab at the Explorer page for your contract's address.
+Navigate to the _Contract_ tab at the Explorer page for your contract's address.
+<img width="1359" alt="contract_tab" src="https://user-images.githubusercontent.com/73849597/144134924-e800ba23-a65d-4adb-ae4a-26f3fe631f6c.png"></img>
```
Comment (yulin-dong) [inline docs/build/tutorials/smart-contracts/verify-smart-contracts.md:12]: @Julian-dev28 does Ava Labs have full control of these images? Can these images be deleted without our knowledge? Can you put these images under `avalanche-docs/static/img` directory?

**Comment by yulin-dong** [comment]
I think I have fixed everything I can to make this PR working with the new Docusaurus. Please check it out https://ba8b5630.avalanche-docs.pages.dev.

@Julian-dev28 please verify the contents and move all images to `avalanche-docs/static/img` from  ` https://user-images.githubusercontent.com/73849597/` and make the proper changes to reference them.

File: docs/build/tutorials/smart-contracts/verify-smart-contracts.md [hunk 2]
```diff
@@ -8,17 +8,20 @@ If you have issues, contact us on [Discord](https://chat.avalabs.org).
...
+
+![compiler](https://user-images.githubusercontent.com/73849597/144135003-e0b98b9e-7f66-40b1-a430-1358616e9b9f.png)
```
Comment (yulin-dong) [inline docs/build/tutorials/smart-contracts/verify-smart-contracts.md:14]: this needs to be changed to local file

File: docs/build/tutorials/smart-contracts/verify-smart-contracts.md (hunk 1 above)
Comment (yulin-dong) [reply docs/build/tutorials/smart-contracts/verify-smart-contracts.md:12]: this needs to be changed to local file

File: docs/build/tutorials/smart-contracts/verify-smart-contracts.md [hunk 3]
```diff
@@ -8,17 +8,20 @@ If you have issues, contact us on [Discord](https://chat.avalabs.org).
...
-![Verify &amp; Publish](/img/smart-contract-verify-page.png)
 
 Click _Verify & Publish_ to enter the smart contract verification page.
 
-![Contract Entry](/img/smart-contract-input-page.png)
 
 [Libraries](https://docs.soliditylang.org/en/v0.8.4/contracts.html?highlight=libraries#libraries) can be provided. If they are, they must be deployed, independently verified and in the _Add Contract Libraries_ section.
 
-![Libraries](/img/smart-contract-library.png)
+![libraries](https://user-images.githubusercontent.com/73849597/144134976-e5a6b981-b006-4d5a-b47c-04bef1978448.png)
```
Comment (yulin-dong) [inline docs/build/tutorials/smart-contracts/verify-smart-contracts.md:22]: this needs to be changed to local file

**Comment by yulin-dong** [comment]
yarn build
yarn run v1.22.15
$ docusaurus build

[en] Creating an optimized production build...

✖ Client
  Compiled with some errors in 9.50s

● Server █████████████████████████ building (64%) 1/1 entries 2250/2250 dependencies 928/928 modules 0 active




Error: Image docs/build/tutorials/smart-contracts/"/img/truffle-verify-txn1.png" used in docs/build/tutorials/smart-contracts/verify-smart-contracts-with-truffle-verify.md not found.
    at async Promise.all (index 0)
Client bundle compiled with errors therefore further build is impossible.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Review by Julian-dev28 — APPROVED** [review]

**Comment by yulin-dong** [comment]
@Julian-dev28 there are still 3 files which need to be changed to local file.

File: docs/build/tutorials/smart-contracts/verify-smart-contracts-with-truffle-verify.md [hunk 4]
```diff
@@ -145,11 +145,11 @@ npx truffle migrate --network fuji
 
 
 You should see the txn activity in your terminal
-![truffle-verify-txn1](/img/truffle-verify-txn1.png) 
+![truffle-verify-txn1](static/img/truffle-verify-txn1.png)
```
Comment (yulin-dong) [inline docs/build/tutorials/smart-contracts/verify-smart-contracts-with-truffle-verify.md:148]: Please revert this commit. They all work as is.

**Review by Julian-dev28 — APPROVED** [review]

**Pull request merged**